File: 프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.java
class Solution {
    public int solution(int chicken) {
        int coupon=chicken;
        int remember=0;
        while(coupon>=10){
      
        int service_chicken=coupon/10;
        remember+=service_chicken;
        coupon=service_chicken+(coupon%10);
        
    }
        return remember;
  }
}

File: 프로그래머스/0/120884. 치킨 쿠폰/치킨 쿠폰.js
function solution(chicken) {
    let Chicken=0;
    let coupon=chicken;
    let serviceC=0;
    while(coupon>=10){
        serviceC=Math.trunc(coupon/10);
        coupon=serviceC+coupon%10
        Chicken+=serviceC;
        
    }
    return Chicken;
}